scripts/generate_ip_env_fw_e2e.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Fetch public IPv4 address
+PUBLIC_IPV4=$(curl -s https://api.ipify.org)
+
+# Fetch public IPv6 address
+PUBLIC_IPV6=$(curl -s https://api64.ipify.org)
+
+# Create .env file in /tmp directory
+ENV_FILE="/tmp/ip_vars.env"
```

Comment:
Would it make sense to store this file in something like `/tmp/linode/ip_vars.env` just to scope it a bit more tightly?